### PR TITLE
Add access token attributes integration test

### PR DESCRIPTION
**Pull request by shashimalcse**
### Purpose
$subject 

### Related Issues
- https://github.com/wso2/product-is/issues/20880

### CODE REVIEW

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCAccessTokenAttributesTestCase.java [hunk 1]
```diff
@@ -0,0 +1,176 @@
+package org.wso2.identity.integration.test.oidc;
```
Comment (Thumimku) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCAccessTokenAttributesTestCase.java:1]: license?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCAccessTokenAttributesTestCase.java [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+                .assertThat()
+                .statusCode(HttpStatus.SC_OK)
+                .body("access_token", notNullValue())
+                .body("refresh_token", notNullValue());
+
+        String accessToken = response.then().extract().path("access_token");
+        refreshToken = response.then().extract().path("refresh_token");
+        Assert.assertNotNull(accessToken, "Access token is null");
+        JWTClaimsSet jwtClaimsSet = SignedJWT.parse(accessToken).getJWTClaimsSet();
+        Assert.assertNotNull(jwtClaimsSet.getClaim("username"), "Invalid is null.");
```
Comment (Thumimku) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCAccessTokenAttributesTestCase.java:110]: Error msg?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCAccessTokenAttributesTestCase.java [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
```
Comment (Thumimku) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCAccessTokenAttributesTestCase.java:84]: Error msg?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCAccessTokenAttributesTestCase.java (hunk 2 above)
Comment (mpmadhavig) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCAccessTokenAttributesTestCase.java]: Resolved.

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCAccessTokenAttributesTestCase.java (hunk 3 above)
Comment (mpmadhavig) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCAccessTokenAttributesTestCase.java]: Resolved.

**Comment by mpmadhavig** [comment]
This effort will be tracked with https://github.com/wso2/product-is/pull/21446

**Pull request closed without merging**